doc/classes/VisualShape2D.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="VisualShape2D" inherits="Node2D" keywords="square, rectangle, circle, oval, triangle, capsule, hexagon" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
+	<brief_description>
+		2D shape-drawing node.
+	</brief_description>
+	<description>
+		[VisualShape2D] is a node that draws common geometric shapes like rectangles, circles, triangles, and capsules. It is very useful to prototype 2D applications, as it needs very little setup without needing a [Texture2D].
+		For more advanced capabilities, see [Polygon2D], [Sprite2D], or [MeshInstance2D]. For custom drawing, see [method CanvasItem._draw] and the [CanvasItem] draw methods.
```

Comment:
I'd also mention that it converts to various nodes.

Reply:
I worked it into a sentence about texture. 
I couldn't find too many other references to this kind of functionality, so I just said the Editor can convert it, I don't know if there is a specific term for those buttons that we want to use.